app/assets/javascripts/admin/addon/mixins/setting-component.js

```diff
@@ -214,8 +214,15 @@ export default Mixin.create({
         this.afterSave();
       }
     } catch (e) {
-      if (e.jqXHR?.responseJSON?.errors) {
-        this.set("validationMessage", e.jqXHR.responseJSON.errors[0]);
+      const json = e.jqXHR?.responseJSON;
+      if (json?.errors) {
+        let errorString = e.jqXHR.responseJSON.errors[0];
```

Comment:
```suggestion
        let errorString = json.errors[0];
```

This can be shorter, right?